src/client-entry.js

```diff
@@ -157,7 +157,6 @@ router.onReady(() => {
 		app.$Progress.finish();
 		// fire pageview
 		app.$fireAsyncPageView(to, from);
-		app.$kvFireQueuedEvents();
```

Comment:
This is a typo, but also unneccessary because of the `app.$fireQueuedEvents()` call in the `app.$setKvAnalyticsData()` callback earlier in this file.